doc/Installation/Install-LibreNMS.md

```diff
@@ -523,6 +521,24 @@ to the file. Run:
 chown librenms:librenms /opt/librenms/config.php
 ```
 
+# Set location to fping6
+
+On the validate screen, <http://librenms.example.com/validate> 
+you might get a message like this:
+
+```
+fping6 location is incorrect or bin not installed.
+```
+
+=== "Ubuntu 20.04 / Debian 10"
+    This is not a known problem in this distro.
+=== "CentOS 8"
+    To fix this: copy the following line to `config.php`:
+
+    ```
+    $config['fping6'] = '/usr/sbin/fping';
```

Comment:
This isn't correct as fping will not ping a specific address family.  (ideally we use -4 and -6 inside LibreNMS)  But we need to evaluate the support of those flags.

Reply:
I often create /usr/local/bin/fping6
```bash
fping -6 $@
```

Reply:
The something in validate.php is evaluating this incorrectly. I worked with what output I got from both ./validate.php and web ui. After inserting this line, the "error" went away.

Reply:
Thank you for the info. Question still remains, will validate.php keep complaining about that binary not being set, or will the "test" be fixed so that this work-around is not needed anymore?

Reply:
Not only will it complain about it, it won't work for devices set to use IPv6 exclusively.
We could try fping -6 if fping6 is not found in the code and not require fping6 to be set.

Reply:
unfortunately we can't even test fpng -6 ::1 because some people disable IPv6 in the kernel...

Reply:
I understand that I should remove the line from config.php and also the instructions from the CentOS docs. We just need to know that people will get this message then? Unless the validation is fixed.